webserver/views/api/v1/core.py

```diff
@@ -109,3 +110,36 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+
+@bp_core.route("/recordings", methods=["GET"])
+def get_recordings():
+    """Returns the low level details for all the recording-ids
+       mentioned in the request with their offset. Offsets are
+       optional. If no offset is given then a default value of
+       0 is used.
+    """
+    recording_ids = request.args.get("ids")
+    recordings = recording_ids.split(";")
+    recording_details = {}
+    if len(recordings) > 200:
+        raise webserver.views.api.exceptions.APIBadRequest("More than 200 recordings not allowed per request")
+    for recording in recordings:
+        recording = str(recording).split(":")
```

Comment:
No its not, Its optional. We are checking first if there are two parameters (id and an offset), if there is no offset we set a default offset of 0.